draft-ietf-taps-interface.md

```diff
@@ -3598,6 +3600,7 @@ of potentially limited scope for alternate path discovery during Connection
 establishment, as well as potential additional information leakage about
 application interest when used with a resolution method (such as DNS without
 TLS) which does not protect query confidentiality.
+Names used with the Transport Services API SHOULD be fully-qualified domain names (FQDNs); not providing an FQDN will result in the Transport Services Implementation needing to resolve using DNS search domains, which might lead to inconsistent or unpredictable behavior.
```

Comment:
/needing to resolve/needing it to be resolved/

Reply:
Side note, this particular wording, from @tfpauly, exists in two places.
I think "needing it to be resolved" makes the sentence structure a little complex... how about:
"needing to use DNS search domains for name resolution" ?

Reply:
I am thinking the whole time whether a "with FQDN" in addition to "with hostname" would be a better solution here. that makes it totally clear whether a programmer expects canonisation or not